src/test/java/edu/ucsb/courses/documents/statistics/SingleCourseSearchTests.java

```diff
@@ -0,0 +1,68 @@
+package edu.ucsb.courses.documents.statistics;
+
+import java.util.List;
```

Comment:
Unused import, we should remove.

Reply:
We removed the unused import

Reply:
The package is needed but all the other unused imports have been removed.